connector/jobrunner/runner.py

```diff
@@ -131,6 +135,21 @@
 _logger = logging.getLogger(__name__)
 
 
+# Unfortunately, it is not possible to extend the Odoo
+# server command line arguments, so we resort to environment variables
+# to configure the runner (channels mostly).
+#
+# On the other hand, the odoo configuration file can be extended at will,
+# so we check it in addition to the environment variables.
+
+
+def _channels():
+    return (
+        os.environ.get('ODOO_CONNECTOR_CHANNELS') or
+        config.misc.get("options-connector", {}).get("channels")
```

Comment:
I agree with @leorochael, we might want to deploy an instance with a configuration file, which is kind of fixed, but just override the configuration for one invocation using the env variable